### DISCUSSION TITLE
nbconvert templates can't be found (M1 Monterey)

### DISCUSSION THREAD

**Discussion by stgm** [Homebrew/discussions#2638] [Everyday usage]
Jupyter has a couple of search paths built-in that seem to be hardcoded to `/usr/local` (and `/usr`).

See https://github.com/jupyter/jupyter_core/blob/2ccb57697fb290737ae2477331e0a5539036b3ac/jupyter_core/paths.py#L129

However, `pip3` installs everything in `/opt/homebrew` as would be expected for a Python installation that lives within Homebrew on a M1.

I haven't found a way to make Jupyter use `/opt/homebrew` other than editing the local copy of `paths.py`. When I do that it works perfectly 👍 But my changes will be overwritten eventually.

Any thoughts on this?

**Comment by SMillerDev** [comment]
You can make an issue in their issue tracker about this. They might want to make it configurable

**Reply by stgm** [reply]
Sure, I'll do that. In the meantime, if anyone has relevant ideas to share, please do. Might be that someone's solved it in a different way (e.g. using environment variables or some magic config stuff for Jupyter). I'd be interested to take that into account when reporting to the Jupyter community.

**Reply by pjledge** [reply]
I just ran into this problem and came to report it - in my case broken functionality of the Jupyter 'Print' and 'Download as' File options.

I ended up bypassing the issue by issuing a "sudo ln -s /opt/homebrew/share/jupyter /usr/local/share/jupyter"
